gnupg/_parsers.py

```diff
@@ -1027,6 +1033,9 @@ def sub(self, args):
         subkey = [args[4], args[11]]
         self.curkey['subkeys'].append(subkey)
 
+    def rvk(self, args):
+        self.curuid == None
```

Comment:
My assumption is that this line is a typo, it should be 
```
 if self.curuid == None:
            return
```

like in the other lines.